src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public interface Figure {
+    void draw();
+
+    double getArea();
+
+    String getColor();
+
```

Comment:
lets divide this interface into 2 - AreaCalculator and Drawable
Interfaces are used to define behaviour